jerry-main/main-unix.c

```diff
@@ -375,6 +375,41 @@ register_js_function (const char *name_p, /**< name of the function */
   jerry_release_value (result_val);
 } /* register_js_function */
 
+/**
+ * Check whether JerryScript has a requested feature enabled or not. If not,
+ * print a warning message.
+ *
+ * @param FEATURE feature to check
+ * @param OPTION command line option that triggered this check
+ * @return the status of the feature.
+ */
+#define CHECK_FEATURE(FEATURE, OPTION) \
+  jerry_is_feature_enabled (FEATURE) \
+  ? true \
+  : (jerry_port_default_set_log_level (JERRY_LOG_LEVEL_WARNING), \
+     jerry_port_log (JERRY_LOG_LEVEL_WARNING, "Ignoring '%s' option because this feature is disabled!\n", (OPTION)), \
+     false)
```

Comment:
unusual style. Please put every parameter to a separate line and indent properly.